src/layout/Menu/Menu.style.ts

```diff
@@ -153,18 +143,54 @@ export const MenuStyled = styled.div<{
 
 export const SubMenu = styled.div<{ total: number }>`
   width: 100%;
-  padding-top: 0.25rem;
 
   > div {
-    animation: ${fade} 120ms ease-in-out both;
+    animation: ${fade} 140ms ease-in-out both;
 
     ${(p) =>
       [...new Array(30)].map(
         (_, i) => css`
-          &:nth-child(${i}) {
-            animation-delay: ${i * (20 + 100 / p.total)}ms;
+          &:nth-child(${i + 1}) {
+            animation-delay: ${(i + 1) * (20 + 100 / p.total)}ms;
           }
         `
       )};
```

Comment:
I know I wrote this, but it's admittedly hacky 🙈. Now that we need to support a very large number of items, a more correct way to do this would be to pass them index from the `SubMenu` children array to `Item` to `SimpleItem` and `ComplexItem` and then add `{ animationDelay: ... }` to the top-level `Wrapper` element in both.